commit 490b4ef3dc813bfbff0667127bd0dd8383dddc08
Author: tblume <Thomas.Blume@suse.com>
Date:   Tue Aug 2 14:55:25 2016 +0200

    yummy:patchX:systemd-ask-password--make-sure-directory-watch-is-started-before-cryptsetup---3850-.patch

systemd-ask-password: make sure directory watch is started before cryptsetup (#3850)

    
    The password directory watch should get ordered before cryptsetup to make sure
    that the password for unlocking the crypt device gets prompted.
    (cherry picked from commit 7633f8ef371a0992374956437fb7bb8189156b10)

diff --git a/units/systemd-ask-password-console.path b/units/systemd-ask-password-console.path
index 2949635..7899ae7 100644
--- a/units/systemd-ask-password-console.path
+++ b/units/systemd-ask-password-console.path
@@ -11,7 +11,7 @@ Documentation=man:systemd-ask-password-console.service(8)
 DefaultDependencies=no
 Conflicts=shutdown.target
 After=plymouth-start.service
-Before=paths.target shutdown.target
+Before=paths.target shutdown.target cryptsetup.target
 ConditionPathExists=!/run/plymouth/pid
 
 [Path]
diff --git a/units/systemd-ask-password-wall.path b/units/systemd-ask-password-wall.path
index 95ec9bc..a3ca617 100644
--- a/units/systemd-ask-password-wall.path
+++ b/units/systemd-ask-password-wall.path
@@ -10,7 +10,7 @@ Description=Forward Password Requests to Wall Directory Watch
 Documentation=man:systemd-ask-password-console.service(8)
 DefaultDependencies=no
 Conflicts=shutdown.target
-Before=paths.target shutdown.target
+Before=paths.target shutdown.target cryptsetup.target
 
 [Path]
 DirectoryNotEmpty=/run/systemd/ask-password
